rfcs/97-add-maximum-file-size-for-storage-write.md

```diff
@@ -0,0 +1,145 @@
+---
+author: JinnyYi <github.com/JinnyYi>
+status: draft
+updated_at: 2021-06-09
+---
+
+# GSP-97: Add Maximum File Size for Storage Write
```

Comment:
Well, I found we can split into two GSPs:

- Add Storage Meta Maximum Size
- Add Storage Meta Service Metadata

Reply:
OK. I'll new another GSP to propose `Add Storage Meta Service Metadata` and modify this one focus on `Add Storage Meta Maximum Size`.

Reply:
After recent changes, how about update the title to `Add Restrictions In Storage Metadata`?

Reply:
> * Add Storage Meta Service Metadata

ref: https://github.com/beyondstorage/specs/pull/111